### ISSUE TITLE
Any plans to update this for MyLogin?

### ISSUE THREAD

**Issue by neilgilmour** [wondeltd/Socialite#3]
I see the demo Laravel app, and was Wondering if there's any plan from Wonde to update this Socialite provider to support MyLogin.

**Comment by lifeontime** [comment]
+1

**Comment by samdharris** [comment]
I don't think this will happen. Wonde SSO & MyLogin are 2 completely separate entities and a Wonde school may not necessarily have MyLogin setup. If this was to happen, it would be a completely different package.
